src/ReportManager/ReportDetailView/index.js

```diff
@@ -727,6 +727,64 @@ const ReportDetailView = ({
 
   const isReadOnly = reportSchemas?.schema?.readonly;
 
+  const efb = {
```

Comment:
We should remove this before merging. Or at least adding some kind of temporal feature flag to use only in development to inject this data.

Reply:
Sure, also I was thinking about adding a FF for better management, also, since We are going to use mock data to tests these form I was thinking on adding a select control, that will allow QA and us to select a mock object to render a form based on the selection, by doing this We can use several configurations of schemas within a feature env. 

Let me finish with all major things regarding this ticket, once that done I will elaborate a little bit more on this idea